### ISSUE TITLE
Can you share pretrained model for discriminator?

### ISSUE THREAD

**Issue by RaymondJiangkw** [marcoamonteiro/pi-GAN#11]
Hey, thank you for your splendid work!
After some tests and diving into your example codes, I am shocked by your model's performance on generating consistent and truthful faces on different views quickly. A Neural Radiance Field representing human faces of diverse situations is reached. Thus, I want to write an encoder to project real faces into w space quickly and faithfully.
However, during training, I found that the discriminator in pi-GAN may help the encoder perform better since it not only outputs whether faces are real or fake, which can be used as an additional adversarial loss term, but also outputs pitch and yaw, which should be used for parameters when rendering. Tilted faces from real dataset seem fail to be encoded well into w space, given h_mean and v_mean are set to pi / 2.
Of course, I could train a discriminator of pi-GAN by myself based on pretrained generators and I am currently doing it, but because of the potential instability of initialization and the cost of time, I will really appreciate it if you could share the pretrained model for the discriminator.
Thank you!

**Comment by ericryanchan** [comment]
Hi Raymond, sorry for taking so long to get back to you! I'll do my best to get you a checkpoint that includes a discriminator by tomorrow :)

**Comment by RaymondJiangkw** [comment]
Oh, thank you so much!

**Comment by ericryanchan** [comment]
Hi Raymond, I've added a drive link with a checkpoint [here](https://drive.google.com/drive/folders/1K3lGcJwgjrrzZEgf_QaRNBc9k9eml0Ke?usp=sharing). I think it may be a different run than the other linked models but hopefully, it'll still work for your purposes! Let me know if you run into any problems loading it :)

**Comment by RaymondJiangkw** [comment]
Hi Eric, I have downloaded the checkpoint and loaded it successfully. Thank you so much for help and your splendid work! 😄

**Comment by lixiaNii** [comment]
Hi Eric, thanks a lot for your awesome work and the resources! I'm doing research about car generation and want to fine-tune your network for tests. Can you also share the pretrained discriminator of CARLA? @ericryanchan

**Comment by Daisy-Zhang** [comment]
Hello~
I am also trying to train a pi-GAN on my own custom face dataset but it costs too much time when training from scratch. I would really appreciate it if you could share the pretrained checkpoints for the discriminator, optimizer, and scaler on CelebA. Thank you very much!
